contracts/cw1-whitelist/src/multitest.rs

```diff
@@ -0,0 +1,116 @@
+use cosmwasm_std::testing::{mock_env, MockApi, MockStorage};
+use cw_multi_test::{App, BankKeeper, Contract, ContractWrapper, Executor};
+use cosmwasm_std::{Addr, Empty};
+use anyhow::Result;
+use cw1::Cw1Contract;
+use derivative::Derivative;
+
+fn mock_app() -> App {
+    let env = mock_env();
+    let api = MockApi::default();
+    let bank = BankKeeper::new();
+    let storage = MockStorage::new();
+
+    App::new(api, env.block, bank, storage)
+}
+
+fn contract_cw1() -> Box<dyn Contract<Empty>> {
+    let contract = ContractWrapper::new(
+        crate::contract::execute,
+        crate::contract::instantiate,
+        crate::contract::query,
+    );
+    Box::new(contract)
+}
+
+/// Configuration of single member
+struct MemberConfig {
```

Comment:
As it has just single field, I would just remove this proxy struct and store `String` as members.